echopype/tests/test_process_ad2cp.py

```diff
@@ -37,3 +38,111 @@ def test_process():
             assert testing_group.equals(base_group), f"process ad2cp failed on {file} with group {group}"
 
         os.remove(tmp_file)
+
+OCEAN_CONTOUR_UNPARSED = [
+    "./echopype/test_data/ad2cp/SG198_0097_a.ad2cp",
+    "./echopype/test_data/ad2cp/SG198_0097_b.ad2cp",
+    "./echopype/test_data/ad2cp/SG198_0099_a.ad2cp",
+    "./echopype/test_data/ad2cp/SG198_0099_b.ad2cp"
+]
+
+OCEAN_CONTOUR_PARSED = [
+    "./echopype/test_data/ad2cp/SG198_0097_a.ad2cp.nc",
+    "./echopype/test_data/ad2cp/SG198_0097_b.ad2cp.nc",
+    "./echopype/test_data/ad2cp/SG198_0099_a.ad2cp.nc",
+    "./echopype/test_data/ad2cp/SG198_0099_b.ad2cp.nc"
+]
+
+def test_ocean_contour():
+    for file, parsed_file in zip(OCEAN_CONTOUR_UNPARSED, OCEAN_CONTOUR_PARSED):
+        test_convert = Convert(file, model="AD2CP")
+        test_file = ".".join(file.split(".")[:-1]) + ".nc"
```

Comment:
I think it'll be great for us to use `pathlib.Path` consistently for our path operations. Could you change this and other similar path ops to use `Path`?